src/app/modules/core/mocks/index.ts

```diff
@@ -85,4 +89,53 @@ export const Mocks = {
     finalizedSlot: 960,
     finalizedEpoch: 30,
   } as ChainHead,
+  '/eth/v1alpha1/node/peers': {
+    peers: [
+      {
+        connectionState: ConnectionState.CONNECTED,
+      },
+      {
+        connectionState: ConnectionState.CONNECTED,
+      },
+      {
+        connectionState: ConnectionState.DISCONNECTED,
+      },
+      {
+        connectionState: ConnectionState.DISCONNECTED,
+      },
+      {
+        connectionState: ConnectionState.CONNECTED,
+      },
+      {
+        connectionState: ConnectionState.CONNECTED,
+      },
+    ] as Peer[]
+  } as Peers,
+  '/eth/v1alpha1/validators/participation': {
+    epoch: 32,
+    finalized: true,
+    participation: {
+      currentEpochActiveGwei: "1446418000000000" as any,
```

Comment:
The reason these use any is because our gRPC gateway returns them as strings but the typescript proto defines them as numbers, which is not a big deal